server/app.py

```diff
@@ -120,6 +116,11 @@ def hello():
     return redirect(url_for("index"))
 
 
+@app.route("/favicon.ico")
+def favicon():
+    return ""
+
+
```

Comment:
Why

Reply:
Saw some Error 404's about accessing the favicon when going to the API routes via the browser. So, I added this route to stop that from happening.

Reply:
Ah, the favicon for the backend pages?